homeassistant/components/nws/services.yaml

```diff
@@ -0,0 +1,13 @@
+get_forecasts_extra:
+  target:
+    entity:
+      domain: weather
```

Comment:
Shouldn't we filter only nws integration entities too?

Reply:
See #122986

Reply:
Thanks!